physlr/physlr.py

```diff
@@ -2195,11 +2195,20 @@ def physlr_path_to_fasta(self):
         num_contigs = 0
         num_bases = 0
 
-        used_seqs = {name[0:-1] for path in paths for name in path if name[-1] != "."}
-
         gaps = "N" * self.args.gap_size
 
         for path in progress(paths):
+            all_unoriented = True
+            longest_seq = 0
+            for idx, name in enumerate(path):
+                if name[-1] != ".":
+                    all_unoriented = False
+                    break
+                if len(name) > len(path[longest_seq]):
+                    longest_seq = idx
+            if all_unoriented:
+                path = None
```

Comment:
What about something like:
```
all_unoriented = all([name[-1] == "." for name in path])
if all_unoriented:
    continue
```
 I don't think you need the length tracking anymore

Reply:
That is correct. Didn't know there there was an `all` function